modules/social_features/social_event/social_event.module

```diff
@@ -1243,3 +1243,11 @@ function social_event_preprocess_fieldset(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_social_core_compatible_content_forms_alter().
+ */
+function social_event_social_core_compatible_content_forms_alter(array &$compatible_content_type_forms) {
```

Comment:
We don't need this hook for core content types, since they already added in the `social_core_form_node_form_alter()`.